doc/source/reference/upgrading.md

```diff
@@ -19,6 +19,13 @@ Seldon Core adds support for Kubernetes 1.22 by upgrading all ValidatingWebhookC
   * Access required to modify files in the local folder are required so the application folder should be writable
   * The default base image now changes the owner of the /microservice folder to user 8888
 
+### Updated executor request logger settings
```

Comment:
Nice - we should make sure this is also covered in the UPGRADING docs in the deploy side